manuscript/1-SoftReq.txt

```diff
@@ -235,14 +235,14 @@
 Συνήθως υπάρχουν πολλοί άνθρωποι που εμπλέκονται, εκτός από τον ειδικό
 απαιτήσεων, καθένας από τους οποίους έχει μερίδιο στο λογισμικό.
 Τα ενδιαφερόμενα μέρη ποικίλλουν μεταξύ των έργων, περιλαμβάνουν όμως
-πάντα τους χρήστες / διαχειριστές και τους πελάτες (που δεν χρειάζεται
+πάντα τους χρήστες/ διαχειριστές και τους πελάτες (που δεν χρειάζεται
```

Comment:
According to the Chicago Manual of Style (5.122) there should be no spaces in any of the two sides of the solidus.